Gopkg.toml

```diff
@@ -55,10 +61,20 @@ required = ["github.com/golang/protobuf/protoc-gen-go"]
   name = "k8s.io/apimachinery"
   version = "kubernetes-1.10.2"
 
+[[override]]
+  name = "k8s.io/code-generator"
+  version = "kubernetes-1.10.2"
+
 [[override]]
   name = "github.com/coreos/etcd"
   version = "3.3.5"
 
 [prune]
   go-tests = true
   unused-packages = true
+
+  [[prune.project]]
+    name = "k8s.io/code-generator"
+    unused-packages = false
+    non-go = false
```

Comment:
non-go = false is redundant and should be removed.

Reply:
I was wondering about this, thanks!